hal/armv7m/stm32/l4/peripherals.h

```diff
@@ -73,4 +73,7 @@
 #define FLASH_PROGRAM_2_ADDR    0x08080000
 #define FLASH_PROGRAM_BANK_SIZE (512 * 1024)
 
+#define RAM_ADDR      0x20000000
+#define RAM_BANK_SIZE (320 * 1024)
```

Comment:
We have place for statically defined memory areas: [armv7m4-stm32l4x6.ldt](https://github.com/phoenix-rtos/plo/blob/master/ld/armv7m4-stm32l4x6.ldt).  Every statically defined memory area/region should be placed there.

Btw. not the case for this PR, but `FLASH_PROGRAM_*` should also be moved to the `*ldt` file.

Reply:
Moved in the last force-push, did You mean sth like that?

Reply:
I agree with @gerard5 that memory areas should be located in linker script.
I know that moving `FLASH_PROGRAM_*` define is not in the scope of this PR but you can do it in a separate commit within this PR.

Reply:
Done in second commit.

Reply:
Yes.